source/bug-report-guidelines.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Bug Report Guidelines
+type: community
+has_chapter_content: false
+version: 2.1
+---
+
+# Bug Report Guidelines
+
+This document describes how to write a good Weex bug report. Good bug reports help developers decide an appropriate priority and severity for a bug, and increase the chance that a bug will be fixed quickly. The more specific information you can provide, the better.
```

Comment:
to classify the priority and severity of a bug properly, which helps the bug get fixed as soon as possible. The more specific information you provide, the better your bug gets understood.